src/options.cpp

```diff
@@ -628,6 +629,75 @@ std::string options_manager::build_tilesets_list()
     return tileset_names;
 }
 
+/** Fill SOUNDPACKS mapping with values.
+ * Scans all directores in FILENAMES["sounddir"] directory for file named FILENAMES["soundpack-conf"].
+ * All founded values added in mapping SOUNDPACKS as name, soundpack_dir.
+ * Furthermore, it builds possible values list for cOpt class.
+ * @return One string containing all found soundpacks in form "soundpack1,soundpack2,soundpack3,..."
+ */
+std::string options_manager::build_soundpacks_list()
```

Comment:
This looks like a copy of the function for the tilesets. Is there so much difference between those two function? They seem to behave almost identical, only using different input / output variables. There should probably be one generic function that is called for both.